### PR TITLE
[IMP] stock_by_warehouse: Add tests to verify the warehouses_stock_recompute functionality T#63809

### PR DESCRIPTION
**Pull request by rolandojduartem**

### CODE REVIEW

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, please, could you review this PR?

cc @desdelinux

File: stock_by_warehouse/tests/test_stock_available_unreserved.py [hunk 1]
```diff
@@ -52,6 +54,7 @@ def setUp(self):
             "name": "product A",
             "default_code": "A",
         }
+        self.env.user.groups_id = [Command.link(self.env.ref("stock.group_stock_multi_warehouses").id)]
```
Comment (luisg123v) [inline stock_by_warehouse/tests/test_stock_available_unreserved.py]: Use `|=`, it's simpler

File: stock_by_warehouse/tests/test_stock_available_unreserved.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-from odoo.tests.common import TransactionCase
+from odoo import Command
+from odoo.tests import Form, TransactionCase
+from odoo.tools.safe_eval import safe_eval
 
 
 class TestStockLogisticsWarehouse(TransactionCase):
```
Comment (luisg123v) [inline stock_by_warehouse/tests/test_stock_available_unreserved.py:5]: setUpClass

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, ready!

**Review by luisg123v — APPROVED** [review]

**Pull request merged**